devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -56,7 +61,11 @@ def parse(String description) {
     if (event) {
         if (event.name == "power") {
             def powerValue
-            powerValue = (event.value as Integer)/10            //TODO: The divisor value needs to be set as part of configuration
+            if (powerDivisor) {
+			    powerValue = (event.value as Integer) / (int) powerDivisor
```

Comment:
you mixed tabs w/ spaces here

If you wanted to convert the whole thing to tabs I'd appreciate it.